src/utils/acquireAppFunctionsProject.ts

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+import path from 'path';
+
+// eslint-disable-next-line consistent-return
+const acquireAppFunctionsProject = (dir: string): string => {
+  const expected = ['functions.json', 'package.json', '.app-functions'];
+  const intersection = fs
+    .readdirSync(dir)
+    .filter(file => expected.includes(file));
+
+  if (intersection.length === expected.length) {
+    return path.basename(dir);
+  }
+
+  console.log('The current directory is not an app functions project. Abort.');
+
+  process.exit();
```

Comment:
I think you can use `throw` with a message instead of exiting the process manually. E.g. 

```javascript
throw new Error("The current directory is not an app functions project. Abort.");
```